web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,162 @@
 sidebar_position: 2
 ---
 
-# Creating a New Project
+## Choosing an Approach
 
-## Prerequisites
+### Use Build Tools if:
+
+- Testing is an important part of your workflow
+
+- You don’t want to worry about manually pushing changes to your GH repo.
+
+### Use Dashboard Upstream if:
+
+- The simplest possible setup
+
+- Pantheon repo is your source of truth
+
+- GitHub is your default VCS and CircleCI is your CI provider then this approach will automatically default to both these options. For other options use the build tools approach
+
+### Installing using Dashboard Upstream
+- Create an empty upstream site:
+
+#### Via the Pantheon Dashboard at this link: 
+- https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65
+
+#### Or Alternatively via Terminus:
+
+```
+terminus site:create my-new-site "Describe Site" empty --org='My Team Name'
+```
+
+Replace `'{My Team Name}'` with your team name - for example `My Agency`. This can also be omitted.
+
+#### In the Pantheon Admin Dashboard:
+
+Switch **Development Mode** to **Git**
+
+Click the **Upgrade** button, and change the site plan to **Performance** > **Medium**. 
+
+** Clone your sites repo on your local
+
+** Change into the newly cloned directory
+
+** Add remote for the decoupled upstream: `git remote add upstream https://github.com/pantheon-systems/decoupled-drupal-composer-managed`
+
+** Checkout the main branch from upstream:
+
+```
+
+git fetch upstream main
+
+git checkout -t upstream/main
+
+```
+
+** Reset the master branch to match main, then force push that to the Dev environment:
+
+```
+
+git branch -D master
+
+git checkout -b master
+
+git push origin master --force
+
+```
+
+:::Note
+Make sure code has synced and all active workflows are complete in the UI.
+Click on the **Visit Development Site** button to Install via the UI—selecting either the Pantheon Decoupled Profile, or Pantheon Decoupled Umami Demo profiles.
+:::
+
+## Installing using Build Tools
+### Prerequisites
 
 - Composer (required for CMS backends): [Install Globally](https://getcomposer.org/download/)
 - [Generate machine token](https://pantheon.io/docs/machine-tokens#create-a-machine-token) & [Authenticate into Terminus](https://pantheon.io/docs/machine-tokens#authenticate-into-terminus)
 - [Install Terminus](https://pantheon.io/docs/terminus/install) (3.0.0 above required)
 - Also install the following plugins:
   - `terminus self:plugin:install terminus-build-tools-plugin`
-  - `terminus self:plugin:install terminus-power-tools`
+  - `terminus self:plugin:remove terminus-power-tools`
   - `terminus self:plugin:install terminus-secrets-plugin`
   - Reload the terminus plugins: `terminus self:plugin:reload`
   - Clear cache for composer: `composer clear-cache`
   - Validate that the required plugins are installed: `terminus self:plugin:list`
 - Create [Github Personal access tokens](https://github.com/settings/tokens)
```

Comment:
No one will need to create all four of these tokens, so we will need to describe this differently. Technically build tools should prompt the user for the appropriate values, so maybe we could explain that? Or possibly there is something in the build tools docs that we could link to instead.

Reply:
I don't think this one is resolved. Rather than listing out the types of tokens, could we instead explain that build tools will prompt for additional information?

Here's how the build tools docs addresses this:

```
This command will create:

A Pantheon site
A GitHub repository
A CircleCI test configuration

It will prompt you for the credentials it needs to create these assets. While GitHub and CircleCI are the defaults, other providers are supported as well. See [available services](https://github.com/pantheon-systems/terminus-build-tools-plugin#available-services) for details.
```